jdaviz/core/template_mixin.py

```diff
@@ -1567,7 +1567,7 @@ class AutoTextFieldMixin(VuetifyTemplate, HubListener):
         :value.sync="label"
         :default="label_default"
         :auto.sync="label_auto"
-        :invalid_msg="invalid_msg"
+        :invalid_msg="label_invalid_msg"
```

Comment:
Ah, so I wasn't going crazy! Thanks for fixing it, Kyle. 😸

Reply:
It was mostly my fault for having incorrect dev docs for the mixin (now fixed) 😉